LovethTee.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "Loveth",
+  "url": "https://github.com/LovethTee",
+  "image": "https://user-images.githubusercontent.com/102753233/181619651-b63b702b-4805-4689-ae46-7110febd5985.jpg",
+  "why": "Tohelp inspire other devs in their fitness journeys",
+  "weekly": [weight lifting 4x a week + stretch post workout, yoga class 1x a week, ], [sleep min 7h a day, go to bed before midnight, decrease screen time, eat protein rich foods 3-4x a day]
```

Comment:
Kindly enclose the text in "" like:
"weekly": ["weight lifting 4x a week", "stretch post workout", "yoga class 1x a week", "sleep min 7h a day", "go to bed before midnight", "decrease screen time", "eat protein rich foods 3-4x a day"]

Reply:
updated. Thank you for pointing out the errors.